src/routes/(index)/components/Dashboard.svelte

```diff
@@ -96,25 +96,26 @@
 
 	<div class="h-12"></div>
 
-	<h2 class="text-2xl font-black text-gray-100 md:text-4xl">Your Workspaces</h2>
+	<h2 class="text-2xl font-black text-black md:text-4xl dark:text-gray-100">Your Workspaces</h2>
 	<div class="h-6"></div>
 
-	<div class="mb-2 font-medium">Show Hidden Files?</div>
+	<div class="mb-2 font-medium text-black dark:text-gray-100">Show Hidden Files?</div>
 	<RadioGroup
 		bind:value={showHidden}
 		options={{
 			yes: 'Yes',
 			no: 'No'
 		}}
+		theme={localStorage.theme ?? 'dark'}
```

Comment:
should this be `userData.theme` instead? `userData.theme` should default to `localStorage.theme` I think?